testnets/cosmwasmtestnet/assetlist.json

```diff
@@ -0,0 +1,40 @@
+{
+  "$schema": "../../assetlist.schema.json",
+  "chain_name": "malaga-420",
```

Comment:
the chain name should match the folder name, so 'cosmwasmtestnet' in this case.

Reply:
looks like this was forgotten. I believe this would be the last place that needs updating

Reply:
@msteiner96 this is the final change needed

Reply:
I'll just merge and fix this one.